docs/fsharp/language-reference/byrefs.md

```diff
@@ -98,7 +98,48 @@ All of these rules together mean that the holder of an `inref` pointer may not m
 
 ### Outref semantics
 
-The purpose of `outref<'T>` is to indicate that the pointer should only be written to. Unexpectedly, `outref<'T>` permits reading the underlying value despite its name. This is for compatibility purposes. Semantically, `outref<'T>` is no different than `byref<'T>`.
+The purpose of `outref<'T>` is to indicate that the pointer should only be written to. Unexpectedly, `outref<'T>` permits reading the underlying value despite its name. This is for compatibility purposes.
+
+Semantically, `outref<'T>` is no different than `byref<'T>`, except for one difference where `outref<'T>` parameters can be implicitly constructed into a tuple with the return value when a tuple-style method can be changed to return a tuple when you only supply non-`outref<'T>` arguments to the method.
+
+```fs
```

Comment:
Don't provide the `byref` example here. Just use the `outref` one. Better to have less code.